engine/src/test/java/org/hibernate/validator/test/predefinedscope/PredefinedScopeValidatorFactoryTest.java

```diff
@@ -104,7 +108,7 @@ public void testValidationOnUnknownBeanMethodReturnValue() throws NoSuchMethodEx
 		assertNoViolations( violations );
 	}
 
-	@Test(expectedExceptions = ValidationException.class, expectedExceptionsMessageRegExp = "HV000249.*")
+	@Test(expectedExceptions = ValidationException.class, expectedExceptionsMessageRegExp = "HV000249:.*")
```

Comment:
updating the other commit where these tests were added would make this commit cleaner :smiley:

Reply:
Done.